gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -181,6 +181,9 @@ const CssVariablesPage = () => {
           <PropertyDocItem propertyVar="--nx-border-default">
             Standard border styles for higher-level elements.
           </PropertyDocItem>
+          <PropertyDocItem propertyVar="--nx-border-secondary">
+            Lighter border styles generally used inside components, or after a default border.
```

Comment:
Do we want to make it explicit in the documentation like this that they are lighter?  I don't think there's anything inherently lighter about them that's just the styling that was chosen.

Reply:
> Do we want to make it explicit in the documentation like this that they are lighter? I don't think there's anything inherently lighter about them that's just the styling that was chosen.

Good point, fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/444/commits/82dc27415b4429cf0d7336cfd3ea0a0e507da291